README.md

```diff
@@ -124,7 +122,59 @@ autohide=autohide
 
 Note: You can use `arrows`, `home`, `end`, `Page up` and `Page down` keys to move around in Zulip-Terminal.
 
-## Setting up a development environment
+## Troubleshooting: Common issues
```

Comment:
We should also add that users can use `Ctrl + L` to clear any screen distortions.

Reply:
@amanagr If this PR is good as it stands, I'm inclined to merge first?

Re `ctrl + l`, I think it would be useful, but isn't this a duplication of the hot-key help? Shouldn't we improve that text in preference to adding a separate section, or do you think both is warranted?